Extensions/retags.js

```diff
@@ -102,14 +105,13 @@ XKit.extensions.retags = {
 				   $target = $element;
 				   url = $target.find('.action').data('post-url');
 			// Activity
-			} else if ($element.hasClass('ui_note')) {
-				if ($element.find('.part_response').length) {
-					$element.addClass('is_response');
-				}
+			} else if ($element.hasClass('activity-notification')) {
 				retagClass = 'is_retags';
 				$target = $element;
-				url = $target.find('.part_glass').attr('href');
-			// dashboard
+				var glass = $($element).find('.activity-notification__glass');
```

Comment:
The href seems to be present for me?
![image](https://cloud.githubusercontent.com/assets/233815/26455717/0708d6d4-4139-11e7-89c2-1036964d745c.png)

we might not need as much changes here.